package/CHANGELOG

```diff
@@ -47,11 +47,15 @@ Enhancements
   * Function for obtaining coordinate axes from numpy trajectories now
     globally available in MDAnalysis.analysis.psa (Issue #438)
   * TPR parser updated for Gromacs 5.0.x and 5.1 (Issue #456)
```

Comment:
Open a new 0.12.1 section — as soon as there's a date, it has been released.

@richardjgowers , we should add to the "making a release guide" that after the release the maintainer
* adds the new release stub to CHANGELOG
* increments the release patch level everywhere from X.Y.Z to X.Y.(Z+1)-dev (the `maintainer/change_release.sh` script needs some fixing because it does not properly handled the `-dev`)